api/middleware.go

```diff
@@ -154,6 +156,44 @@ func (a *API) limitHandler(lmt *limiter.Limiter) middlewareHandler {
 	}
 }
 
+func (a *API) limitEmailSentHandler() middlewareHandler {
```

Comment:
I wonder if this should just be a generic rate limiter which you can apply to _any_ route? (eg, why call it `limitEmailSent` and not `rateLimitRoute`), and therefore it shouldn't really matter about `put` and `post`. Not critical for now

Also would be easier to "inject" the config. Then it is a pure function which can be used universally

Reply:
This are not critical for this fix btw - better to get this out asap